lmdeploy/pytorch/configurations/internvl.py

```diff
@@ -1,8 +1,43 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-from .builder import AutoModelConfigBuilder, ProxyAutoModel
+from .builder import AutoModelConfigBuilder
 from .default import DefaultModelConfigBuilder
 
 
+class ProxyAutoModel:
```

Comment:
What's the motivation of this class?
Can we update the `class ProxyAutoModel` in `buidler.py` instead of defining a new one?